runtime/server_header.go

```diff
@@ -33,7 +33,14 @@ import (
 
 // Header defines methods on ServerHeaders
 type Header interface {
+	// Get mirrors the implementation of http.header and returns a single header value.
+	// When a key contains multiple values, -only- the first one is returned.
+	// Ref: https://golang.org/pkg/net/http/#Header.Get
 	Get(key string) (string, bool)
+	// Values mirrors the implementation of http.header and returns a slice of header values.
+	// When a key contains multiple values, the entire collection is returned.
+	// Ref: https://golang.org/pkg/net/http/#Header.Values
+	//Values(key string) ([]string, bool)
```

Comment:
Forgot to uncomment?

Reply:
Yep, that was from a stale commit. Updates pending.